build_rules.gradle

```diff
@@ -621,6 +629,20 @@ artifactId=${project.name}
     }
     artifacts.archives javadocJar
 
+    // Run tests on artifacts
```

Comment:
Update comment to: `// Use a configuration and dependency set which represents the execution classpath using shaded artifacts for tests.`

Reply:
Comment updated.